.svelte-kit/tsconfig.json

```diff
@@ -24,22 +24,21 @@
 			"..",
 			"./types"
 		],
-		"importsNotUsedAsValues": "error",
+		"verbatimModuleSyntax": true,
 		"isolatedModules": true,
-		"preserveValueImports": true,
 		"lib": [
 			"esnext",
 			"DOM",
 			"DOM.Iterable"
 		],
-		"moduleResolution": "node",
+		"moduleResolution": "bundler",
```

Comment:
This change will now break until colorjs has a new release that includes https://github.com/color-js/color.js/pull/333